src/pages/Home/index.js

```diff
@@ -2,25 +2,40 @@ import React, { useState, useEffect, useContext } from "react";
 import { SearchEngine } from "../../components/SearchEngine";
 import { Filter } from "../../components/Filter";
 import { FilmCard } from "../../components/FilmCard";
+import { UserSessionValidation } from "../../components/UserSessionValidation";
 import axios from "axios";
 import { Menu } from "../../components/Menu";
+import { Link, Navigate } from "react-router-dom";
 import "./Home.scss";
+// Context
 import { AppContext } from "../../context/AppContext";
-import { Link } from "react-router-dom";
 
 function Home() {
   // fetch Data
   const [films, setFilms] = useState([]);
-  useEffect(async () => {
-    const response = await axios.get("https://ghibliapi.herokuapp.com/films");
-    setFilms(response.data);
+
+  useEffect(() => {
+    let isSubscribed = true;
+    axios
+      .get("https://ghibliapi.herokuapp.com/films")
```

Comment:
:v duda dudosa, donde es que la API te solicita el access_token para hacer una request? :v, es decir, si, a esto el usuario tiene acceso, pero como sabes que no es un usuario baneado xd o algo asi, con el isSuscribed?, en ese caso esta harcodeado?

Reply:
o el navegador ya sabe acceder automaticamente al token? :v

Reply:
Yes as @LilianaRestrepoTorres said. That piece of code is fetching only the movies and is not our API as well.
The variable isSubscribed was an attempt to fix a warning in home page.